rental.rb

```diff
@@ -1,14 +1,12 @@
 class Rental
-  attr_accessor :date
-  attr_reader :book, :person
+  attr_accessor :date, :person, :book
 
   def initialize(date, person, book)
     @date = date
-
     @person = person
-    person.rentals << self
-
     @book = book
-    book.rentals << self
+
+    @person.add_rental(self)
+    @book.add_rental(self)
   end
```

Comment:
- [ ] When I try to create a rental I get this error message (see image). You can fix this by changing this `initialize` method to this as it was:

  ```
  def initialize(date, person, book)
    @date = date
    @person = person
    @book = book

    person.rentals << self
    book.rentals << self
  end
  ```

  ![image](https://user-images.githubusercontent.com/36542986/213402725-deb82e12-f2d7-4b88-98b6-3adf3913bfcf.png)